rc/tutorial/en/meta-help.md

```diff
@@ -0,0 +1,23 @@
+This is not the help you're looking for
+=======================================
+
+Kakoune doesn't implement an extensive/extensible help system as, say, vim
+does. This is not (only) lazyness, it's on purpose.
+
+Kakoune emphasis on these central principles:
+
+    1. Keep the core as simple as possible
+    2. Give direct feedback to the user (with the help box)
+    3. Incite the user to create and adapt their own tools
+    
+Thus, the main documentation should remain a single web page : https://github.com/mawww/kakoune/blob/master/README.asciidoc
```

Comment:
Not sure about that, we do have the `:doc` command thats expected to replace the README.